src/Pages/GradeSelect/FilterSelector/index.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+export default function FilterSelector(props) {
+  return (
+    <div>
+      <FilterBar>
+        <FilterLists>
+          <FilterList>
+            <form>
+              <FilterBtn type="button" onClick={props.toggleDropdown}>
+                추천순 정렬
+              </FilterBtn>
+              <FilterModal show={props.isDropdownActive}>
+                <FilterFormModal>
+                  <FilterTitle>정렬 기준 선택</FilterTitle>
+                  <FilterPara>
+                    <FIlterOption>
+                      <label>
+                        <input type="radio"></input>출발 시간 순
```

Comment:
```suggestion
                        <input type="radio" />출발 시간 순
```
셀프 클로징 태그 적용할 수 있습니다!

Reply:
수정하였습니다!